src/effects/native/reverbeffect.cpp

```diff
@@ -26,7 +26,7 @@ EffectManifest ReverbEffect::getManifest() {
     EffectManifestParameter* time = manifest.addParameter();
     time->setId("bandwidth");
     time->setName(QObject::tr("Bandwidth"));
-    time->setName(QObject::tr("Band"));
+    time->setShortName(QObject::tr("BW"));
```

Comment:
(by "it" here I mean the problem that "BW" is not a common abbreviation)

Do these annotations actually make it into transifex?
http://doc.qt.io/qt-4.8/i18n-source-translation.html#translator-comments